src/vs/workbench/contrib/testing/browser/testingExplorerView.ts

```diff
@@ -351,9 +352,19 @@ export class TestingExplorerView extends ViewPane {
 			}
 		}
 
-		// If there's only one group, don't add a heading for it in the dropdown.
-		if (participatingGroups === 1) {
-			profileActions.shift();
+		const menuActions: IAction[] = [];
+
+		const key = this.contextKeyService.createOverlay([]);
```

Comment:
One key we probably should have in the overlap is the current group (run/debug/coverage) for extensions who want different behavior

Reply:
Hi, sorry could you clarify a bit what you mean here? There should be a context key for the current group and have it only show for some or make that configurable for the extension author?

Reply:
The menu logic can run for all of them, but an author might want to only have a menu visible in one group, or have different menu entries for each group. So letting them do something like `profileKind == "run"` would be good.